website/docs/prysm-usage/monitoring/grafana-dashboard.md

```diff
@@ -120,8 +120,8 @@ A green notification saying “Datasource updated” should now be visible on th
 ## Creating and importing dashboards
 
 1. The dashboard can now be customised to the users preferences. There are two examples that can be used:
-- [dashboard designed for small amount of validator keys](/assets/grafana-dashboards/small_amount_validators.json)
-- [dashboard designed for more than 10 validator keys](/assets/grafana-dashboards/big_amount_validators.json)
+- [dashboard designed for small amount of validator keys](http://localhost:3000/assets/grafana-dashboards/small_amount_validators.json)
+- [dashboard designed for more than 10 validator keys](http://localhost:3000/assets/grafana-dashboards/big_amount_validators.json)
```

Comment:
Having a relative path to `json` files breaks the docusaurus build for some reason.